exp-player/addon/components/exp-free-response/component.js

```diff
@@ -132,6 +132,15 @@ export default ExpFrameBaseComponent.extend(Validations, {
         }
     },
     actions: {
+        onOpen(obj) {
+            this.obj = obj;
+            Ember.run.later(this, function() {
+                let itemList = document.getElementById('ember-power-select-options-' + this.obj.uniqueId);
```

Comment:
This searches the entire document; you can use `this.$()` to get a selector relative to only this component.